src/Badges.sol

```diff
@@ -270,6 +273,28 @@ contract Badges is
     return mint(msg.sender, _uri, raftTokenId);
   }
 
+  function airdropWithConsent(
+    address _recipient,
+    string calldata _uri,
+    bytes calldata _issuerSignature,
+    bytes calldata _recipientSignature
+  ) external virtual {
+    uint256 raftTokenId = specDataHolder.getRaftTokenId(_uri);
+    require(
+      specDataHolder.isAuthorizedAdmin(raftTokenId, msg.sender),
```

Comment:
This doesn't match the requirements. The sender of this txn can be anyone. Please see ticket.